packages/react/src/React.js

```diff
@@ -25,6 +25,12 @@ if (__DEV__) {
   cloneElement = ReactElementValidator.cloneElement;
 }
 
+const REACT_FRAGMENT_TYPE =
+  (typeof Symbol === 'function' &&
+    Symbol.for &&
+    Symbol.for('react.fragment')) ||
+  0xeacb;
```

Comment:
@clemmy Why is the fallback here a number instead of a string? Not sure I follow.
I'm curious so I can accurately update the react typings for TypeScript, and it seems weird to have the first argument to createElement be a string or a symbol or a number.

Reply:
(sorry, didn't mean to add myself as a reviewer)

Reply:
The fallback here is a couple of reasons:

- Strings are currently used for specifying host components in `React.createElement`, it would be a bit confusing to special case fragments into this space. See https://github.com/facebook/react/pull/10783#discussion_r145575197
- Number comparison is faster than string comparison in JS

Reply:
Sounds good. Thanks!